AppCenterData/AppCenterData/Internal/MSDocumentWrapperInternal.h

```diff
@@ -27,18 +26,55 @@
                                      eTag:(NSString *)eTag
                           lastUpdatedDate:(NSDate *)lastUpdatedDate
                          pendingOperation:(NSString *)pendingOperation
-                                    error:(MSDataError *)error
                           fromDeviceCache:(BOOL)fromDeviceCache;
 
+/**
+ * Initialize a `MSDocumentWrapper` instance.
+ *
+ * @param deserializedValue The document value. Must conform to MSSerializableDocument protocol.
+ * @param jsonValue The document's JSON representation.
+ * @param partition Partition key.
+ * @param documentId Document id.
+ * @param eTag Document eTag.
+ * @param lastUpdatedDate Last updated date of the document.
+ * @param pendingOperation The name of the pending operation, or nil.
+ * @param fromDeviceCache Flag indicating if the document wrapper was retrieved remotely or not.
+ * @param error Document error.
+ *
+ * @return A new `MSDocumentWrapper` instance.
+ */
+- (instancetype)initWithDeserializedValue:(id<MSSerializableDocument>)deserializedValue
```

Comment:
Is this initializer needed? When there's an error, one of the initializers below should suffice.

Reply:
Good point, will do.